docs/resources/interface.md.erb

```diff
@@ -30,61 +31,71 @@ An `interface` resource block declares network interface properties to be tested
       it { should be_up }
       its('speed') { should eq 1000 }
       its('name') { should eq eth0 }
+      its('ipv4_addresses') { should include '10.0.0.5' }
     end
 
 <br>
 
 ## Properties
 
-`name`, `speed`
+`ipv4_address`, `ipv4_addresses`, `ipv4_addresses_netmask`, `ipv4_cidrs`, `ipv6_addresses`, `ipv6_cidrs`, `name`, `speed`
 
-<br>
 
 ## Resource Property Examples
 
-### name
-
-The `name` property tests if the named network interface exists:
+### ipv4_address
 
-    its('name') { should eq eth0 }
+Provided for compatibility with ServerSpec. Returns the first `ipv4_cidrs` entry.
 
-### speed
-
-The `speed` property tests the speed of the network interface, in MB/sec:
-
-    its('speed') { should eq 1000 }
+    its('ipv4_address') { should eq '10.0.0.5/8' }
 
 ### ipv4_addresses
 
-The `ipv4_addresses` property tests if the specified address exists on the named network interface:
+The `ipv4_addresses` property returns an Array of IPv4 addresses as Strings. You may then test if the specified address exists on the named network interface:
 
     its('ipv4_addresses') { should include '127.0.0.1' }
```

Comment:
@clintoncwolfe What type of interface (e.g. bash) is this supposed to be? Or is this supposed to be for a yml file?

Reply:
Network interface - a connection from a computer to a network. Could be wifi, ethernet, fiber optic, or something more exotic.